packages/editor/src/components/post-author/check.js

```diff
@@ -11,6 +11,15 @@ import PostTypeSupportCheck from '../post-type-support-check';
 import { store as editorStore } from '../../store';
 import { AUTHORS_QUERY } from './constants';
 
+/**
+ * Component to render its children only if the post type supports the author.
```

Comment:
I'm not native speaker but I think we can improve the wording a bit here. What about 
```suggestion
 * Wrapper component that renders its children only if the post type supports the author.
```